jiant/tasks/evaluate/core.py

```diff
@@ -1,26 +1,116 @@
 import itertools
 import json
+
 from dataclasses import dataclass
 
 import numpy as np
 import pandas as pd
 import seqeval.metrics as seqeval_metrics
 import torch
-from sklearn.metrics import f1_score, matthews_corrcoef
-from scipy.stats import pearsonr, spearmanr
-from typing import Dict, List
+
+from scipy.stats import pearsonr
+from scipy.stats import spearmanr
+from sklearn.metrics import f1_score
+from sklearn.metrics import matthews_corrcoef
+from typing import Dict
+from typing import List
 
 import jiant.shared.model_resolution as model_resolution
-import jiant.tasks as tasks
-import jiant.tasks.lib.templates.squad_style.core as squad_style
-import jiant.tasks.lib.templates.squad_style.utils as squad_style_utils
-import jiant.tasks.lib.mlqa as mlqa_lib
+
 import jiant.tasks.lib.bucc2018 as bucc2018_lib
+import jiant.tasks.lib.mlqa as mlqa_lib
 import jiant.tasks.lib.tatoeba as tatoeba_lib
+import jiant.tasks.lib.templates.squad_style.core as squad_style
+import jiant.tasks.lib.templates.squad_style.utils as squad_style_utils
+
+from jiant.tasks.retrieval import AbductiveNliTask
```

Comment:
As discussed, could we remove the individual imports and just do `import jiant.tasks.retrieval as tasks retrieval`?

Reply:
Switched to this style of import.